src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5157,6 +5255,283 @@ namespace AirflowNetworkBalanceManager {
 		return CalcWindPressure;
 	}
 
+	Real64
+	AirDensity(
```

Comment:
@jasondegraw do you prefer I use ```PsyRhoAirFnPbTdbW``` instead? It shouldn't be too much work to change it over.